project/settings.py

```diff
@@ -161,4 +162,6 @@
 
 CORS_ALLOWED_ORIGINS = [
     "https://app-loader.jasonhubbard.dev",
-]
\ No newline at end of file
+]
```

Comment:
I thought you wanted to keep all the inherited custom settings in `settings_production` and leave this file as the default Django settings?

Reply:
You might also wanted to disable the `GA_KEY` in non-production deployments, or have different properties per environment.